lib/project-data-updater.js

```diff
@@ -44,9 +44,7 @@ function ProjectDataUpdater(config, repository, updateLock, done) {
 
   var that = this;
   this.done = done || function(err) {
-    if (err) {
-      console.log(err);
-    } else {
+    if (err === null) {
```

Comment:
Yeah, so per my later comment, we can have `function(err, noUpdate)` (or some other parameter name) here. `noUpdate` will only be defined as `true` if the webhook didn't trigger an update, so checking for `noUpdate === true` will help differentiate between "updated without error" and "no update, no error".